### PR TITLE
fix addr alignment in witness

### PR DESCRIPTION
**Pull request by jsign**
This PR fixes a problem in EXTCODESIZE when the slot value had a value <20 bytes.

### CODE REVIEW

File: core/state/access_witness.go
Comment (jsign) [inline core/state/access_witness.go:247]: If `len(addr)<20` this was doing the copy incorrectly.

Another option for this particular bug case is **not** doing this code line change, and do:
image

But I think this proposed change is more general, and if we have a similar situation in other place the current fix would cover it.

Open to opinions though.

**Review by g11tech — APPROVED** [review]
lgtm

**Review by gballet — APPROVED** [review]
I have overthought it way too much, LGTM. Let's not merge it until we are ready to relaunch the testnet.

**Pull request merged**
